.erb-linters/erblint-github.rb

```diff
@@ -0,0 +1 @@
+require "erblint-github/linters"
```

Comment:
I don't believe there's currently a better way to inherit configs or rules from another gem in ERBLint

Related issue: https://github.com/Shopify/erb-lint/issues/317.